.bazelci/presubmit.yml

```diff
@@ -12,12 +12,16 @@ tasks:
     shell_commands:
       - "sudo apt -y update && sudo apt -y install libgmp-dev"
     build_flags:
+      # disable bzlmod explicitly, Bazel flipped the default to `true`
+      - "--noenable_bzlmod"
```

Comment:
Unfortunately, this fix will not work since CI is running `bazel --nosystem_rc --nohome_rc info` before the build even starts and thus the build_flags are not taken into account yet. We would have to add this to our own bazelrc instead.